.github/workflows/ci-image.yml

```diff
@@ -8,4 +8,4 @@ jobs:
         uses: actions/checkout@v3.1.0
       - run: |
           echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
-          docker buildx build --provenance=false --platform linux/amd64,linux/arm64 -t ghcr.io/${GITHUB_REPOSITORY}:ci --push .github
+          docker buildx build --provenance=false --platform linux/amd64,linux/arm64 -t ghcr.io/${GITHUB_REPOSITORY}:ci-2 --push .github
```

Comment:
I'm now wondering whether the number counting up is necessary - we could just use the commit (short) hash or something like that?

Reply:
I still think it is useful to keep the tag, as it gives some idea what the image is.

Reply:
Whether we should use the tag or hash (image hash, not the commit) is another question, I guess.

Reply:
Ah now I get what you mean, my bad. That's a good idea